### ISSUE TITLE
Fix the No attribute'tagname' for NoneType object issue for test : tests.foreman.ui.test_operatingsys.OperatingSys.test_update_os_template

### ISSUE THREAD

**Issue by jyejare** [SatelliteQE/robottelo#2044]
The error throws due to no template displayed in templates tab for os because while creating a os, the template is associated with any organisation.

StackTrace:

```
Traceback (most recent call last):
  File "/home/jyejare/Desktop/RedHat/RobotteloTop/robottelo/tests/foreman/ui/test_operatingsys.py", line 396, in test_update_os_template
    self.operatingsys.update(os_name, template=template_name)
  File "/home/jyejare/Desktop/RedHat/RobotteloTop/robottelo/robottelo/ui/operatingsys.py", line 142, in update
    medium_list=new_mediums)
  File "/home/jyejare/Desktop/RedHat/RobotteloTop/robottelo/robottelo/ui/operatingsys.py", line 62, in _configure_os
    ["operatingsys.template"]
  File "/usr/lib/python2.7/site-packages/selenium/webdriver/support/select.py", line 34, in __init__
    if webelement.tag_name.lower() != "select":
AttributeError: 'NoneType' object has no attribute 'tag_name'
-------------------- >> begin captured logging << --------------------
py.warnings: WARNING: /usr/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:79: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning

py.warnings: WARNING: /usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:769: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

py.warnings: WARNING: /usr/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:79: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning

py.warnings: WARNING: /usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:769: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

root: DEBUG: NoSuchElementException: Could not locate element //div[contains(@class, 'jnotify-notification-error')].
root: DEBUG: NoSuchElementException: Could not locate element operatingsystem_os_default_templates_attributes_0_config_template_id.
--------------------- >> end captured logging << ---------------------
```

**Comment by sghai** [comment]
Fixed in PR #2056

**Comment by elyezer** [comment]
#2056 got merged, closing this.
